service/http/docs/dataflow.rst

```diff
@@ -0,0 +1,40 @@
+Dataflow
+========
+
+HttpChannelConfig
+-----------------
+    - ``asynchronous: bool`` 
+        Unused right now but will be accessible over a websocket in the future 
```

Comment:
```suggestion
  - Unused right now but will be accessible over a websocket in the future 
```

Reply:
Let's follow the format from https://intel.github.io/dffml/master/quickstart/model.html#quickstart-command-line-flags-explained again